ledger/ledger-api-bench-tool/src/main/scala/com/daml/ledger/api/benchtool/submission/FooCommandGenerator.scala

```diff
@@ -127,11 +136,18 @@ final class FooCommandGenerator(
         } else None
       )
     val consumingExerciseO: Option[Command] = consumingPayloadO.map { payload =>
+      val selectedActiveFooContractKey = {
```

Comment:
Maybe a comment explaining when these two statements will not result in the same key.

Reply:
Ok